tests/integration/api_v3/rpc/tokenIDs.test.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+
+const {
+	request,
+} = require('../../../helpers/socketIoRpcRequest');
+
+const {
+	jsonRpcEnvelopeSchema,
+} = require('../../../schemas/rpcGenerics.schema');
+
+const {
+	tokenIDsSchema,
+	tokenIDsMetaSchema,
+} = require('../../../schemas/api_v3/tokenIDs.schema');
+
+const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
+const getTokensIds = async (params) => request(wsRpcUrl, 'get.token.ids', params);
+
+describe('get.token.ids', () => {
+	beforeAll(async () => {
+	});
+
+	it('returns tokens info when call with address', async () => {
+		const response = await getTokensIds({});
+		expect(response).toMap(jsonRpcEnvelopeSchema);
+		const { result } = response;
+		expect(result.data).toMap(tokenIDsSchema);
+		expect(result.meta).toMap(tokenIDsMetaSchema);
```

Comment:
Checking the `goodRequestSchemaFortokenIDs` would already validate both.

```suggestion
		expect(result.data).toMap(goodRequestSchemaFortokenIDs);
```